src/components/layout/navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
+
+function Navbar() {
+  return (
+    <header className="nav">
+      <div className="nav-one">
+        <h1>BookStore CMS</h1>
+        <p>BOOK</p>
+        <p>CATEGORY</p>
+      </div>
+      <div className="user-icon">
```

Comment:
- [ ] It would be better if instead of just the text `BOOK` and `CATEGORY` you create those two links to navigate in your app. Kindly fix this minor issue. 

![Capture](https://user-images.githubusercontent.com/64173189/159980257-b58e4106-b1e5-4708-9824-7abb3822873d.JPG)